app/views/shared/_map.html.erb

```diff
@@ -17,7 +17,9 @@
       </turbo-frame>
     </div>
     <%= render "shared/mapbox" %>
-    <%= link_to "https://mapzy.io/?source=embedded", class: "absolute bottom-2 left-2 z-10", target: "_blank" do %>
-      <%= image_tag "mapzy_logo_text_side.png", class: "h-[1.7rem]" %>
+
+    <%= link_to "https://mapzy.io/?source=embedded", class: "absolute bottom-2 left-2 z-0", target: "_blank" do %>
+      <div class="h-[1.4rem] w-24 md:h-[1.6rem] bg-contain bg-no-repeat bg-[url('logo_white_border_sidetext_transparent.svg')] hover:bg-[url('logo_sidetext_transparent.svg')]">
+      </div>
```

Comment:
The reason why the hover effect does not always work I think is: the <a> container wraps the <div> container, which has the background images. I think if you flip the order of these two, it should work better:

```
  <div class="bg-... hover:bg-..." ...>
     <a></a>
   </div>
```